src/multipart.jl

```diff
@@ -125,9 +125,6 @@ end
 content_type(f::Form) = "Content-Type" =>
                         "multipart/form-data; boundary=$(f.boundary)"
 
+# To be deprecated in HTTP@0.9
```

Comment:
Can we log an issue for this so it's not forgotten?

Reply:
Thats why I conditioned the tests on the version; when the version is bumped CI will fail and then someone has to enable this deprecation.